source/agora/consensus/Ledger.d

```diff
@@ -1260,10 +1291,10 @@ public class Ledger
         return null;
     }
 
-    /// return the last payment block before the current block
-    public Height getLastPaymentBlock () const scope @safe @nogc nothrow pure
+    /// return the last paid out block before the current block
+    public Height getLastPaidHeight () const scope @safe @nogc nothrow pure
     {
-        return lastPaymentBlock(this.getBlockHeight, this.params.PayoutPeriod);
+        return lastPaidHeight(this.getBlockHeight, this.params.PayoutPeriod);
```

Comment:
The rename is welcome, but it clutters the diff and makes reviewing harder, that's why having it in its own commit would be best. My litmus test for such a commit separation is: "Will this feature work without this diff?".

Reply:
I'll split it out into a separate commit.